crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -81,6 +84,12 @@ pub use parallel_scope::*;
 pub struct Commands<'w, 's> {
```

Comment:
We  need docs on `Commands` about how to set the error handler for the specific instance of this.